pkg/ipamd/datastore/data_store.go

```diff
@@ -122,6 +122,19 @@ var (
 		},
 		[]string{"cidr"},
 	)
+	noAvailableAddrs = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "awscni_err_no_avail_addrs",
+			Help: "The number of IP/Prefix assignments that fail due to no available addresses at the ENI level",
```

Comment:
nit on wording, can you change this to:
"The number of pod IP assignments that fail due to no available IP addresses"